packages/waf.rb

```diff
@@ -21,7 +21,7 @@ class Waf < Package
   })
 
   depends_on 'help2man'
-  depends_on 'python27' unless File.exists? '/usr/local/bin/python'
+  depends_on 'python27' unless File.exists? '#{CREW_PREFIX}/bin/python'
```

Comment:
This needs to be in double quotes or the constant is not expanded.

Reply:
Yeah.